backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceSupportedDisease.java

```diff
@@ -23,4 +23,11 @@ public class DeviceSupportedDisease {
   @Id
   @Column(name = "supported_disease_id", nullable = false)
   private UUID supportedDiseaseId;
+
+  public DeviceSupportedDisease(UUID deviceTypeId, UUID supportedDiseaseId) {
```

Comment:
Let's save these backend changes for a later PR - the way our rollbacks work, if we needed to roll back this PR for any reason we'd need to do it twice (once for the backend changes and once for the database changes). That risks breaking the backend, if we roll back the database changes but not the backend ones.